base/applications/utilman/umandlg/umandlg.h

```diff
@@ -0,0 +1,121 @@
+/*
+ * PROJECT:         ReactOS Utility Manager Resources DLL (UManDlg.dll)
+ * LICENSE:         GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:         DLL pre-compiled header file
```

Comment:
No, this is not a pre-compiled header file since it is not registered as such (no `add_pch`).